RELEASE_NOTES.md

```diff
@@ -14,6 +14,29 @@
 
 * The `ExponentialBackoff` and `LinearBackoff` classes now require keyword arguments for their constructor. This change was made to make the classes easier to use and to avoid confusion with the order of the arguments.
 
+* The `BaseApiClient` class is not generic anymore, and doesn't take a function to create the stub. Instead, subclasses should create their own stub right after calling the parent constructor. This enables subclasses to cast the stub to the generated `XxxAsyncStub` class, which have proper `async` type hints. To convert you client:
+
+   ```python
+   # Old
+   class MyApiClient(BaseApiClient[MyServiceStub]):
+       def __init__(self, server_url: str, *, ...) -> None:
+           super().__init__(server_url, MyServiceStub, ...)
+           ...
+
+   # New
+   class MyApiClient(BaseApiClient):
+       def __init__(self, server_url: str, *, ...) -> None:
+           super().__init__(server_url, connect=connect)
+           self._stub = cast(MyServiceAsyncStub, MyServiceStub(self.channel))
+           ...
+
+       @property
+       def stub(self) -> ExampleAsyncStub:
```

Comment:
To be more clear,  should this property return `MyServiceAsyncStub` instead?
```suggestion
       def stub(self) -> ExampleAsyncStub:
```

Reply:
Yeah, copy&pasta error, thanks!

Reply:
Updated.